alvr/sockets/src/lib.rs

```diff
@@ -1,12 +1,15 @@
-mod backend;
 mod control_socket;
 mod stream_socket;
 
-use alvr_common::{anyhow::Result, info};
+use alvr_common::{ConResult, HandleTryAgain, anyhow::Result, info};
 use alvr_session::{DscpTos, SocketBufferSize};
 use socket2::Socket;
+#[cfg(not(windows))]
+use std::ffi::c_int;
```

Comment:
This shouldn't be conditional.

Reply:
not sure why that happened, but clippy already caught that